CMakeLists.txt

```diff
@@ -7,11 +7,22 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_BUILD_TYPE Release)
 
 # Avoid building tooling we won't need for release
-set(BUILD_BENCHMARKS OFF CACHE BOOL "" FORCE)
-set(BUILD_FILTERS OFF CACHE BOOL "" FORCE)
-set(BUILD_GENERATORS OFF CACHE BOOL "" FORCE)
-set(BUILD_TESTING OFF CACHE BOOL "" FORCE)
-set(ENABLE_DOCS OFF CACHE BOOL "" FORCE)
+# See all options with `cmake -LA` in an `h3/build` directory,
+# or at https://h3geo.org/docs/next/core-library/compilation-options/
+macro(turn_off option_name)
+    set(${option_name} OFF CACHE BOOL "" FORCE)
+endmacro()
+turn_off(BUILD_ALLOC_TESTS)
```

Comment:
BUILD_ALLOC_TESTS should not have any effect when BUILD_TESTING is off. Harmless to disable it though.

Reply:
I had a suspicion that that was true, but yeah, figured it couldn't hurt to just disable everything to make sure.